lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
```

Comment:
I read this as equivalent to 'whadya' (what did you), though I doubt there's a proper definitions

Reply:
I think it's contextual. Whatcha doin' whatcha got whatchamacallit

Reply:
Ah, hadn't considered that first context. All of these are loosely-defined enough that I think either will yield more readable results (even if they are grammatically incorrect sometimes